components/infobox/wikis/worldoftanks/infobox_league_custom.lua

```diff
@@ -65,6 +73,12 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
+---@param args table
+function CustomLeague:defineCustomPageVariables(args)
```

Comment:
kick this function!
- patch wiki var is deprecated, use the standardized tournament_pach and tournament_endpatch wiki vars instead
- commons already sets the standardized ones already

Reply:
Have you got an example of this in use please?

Reply:
I think this is resolved?

Reply:
counter question: where is `patch` wiki variable used?